onnxruntime/contrib_ops/rocm/bert/skip_layer_norm_tunable_op.h

```diff
@@ -76,75 +76,40 @@ template <typename T>
 Status SkipLayerNormStaticSelection(const SkipLayerNormParams<T>* params) {
   bool hasBias = (params->bias == nullptr) ? false : true;
   bool hasSkipInputBiasAdditionOutput = (params->skip_input_bias_add_output == nullptr) ? false : true;
+  const int grid_size = params->element_count / params->ld;
+  const int block_size = 256;
+
+#define LAUNCH_SKIPLAYERNORM_SMALL_FORWARD(ELEMENTS, TPB, ILP)                              \
+  if (params->ld <= ELEMENTS) {                                                             \
+    SkipLayerNormKernelSmall<T, TPB, ILP><<<grid_size, TPB, 0, params->stream>>>(           \
+        params->ld, params->input, params->skip, params->beta, params->gamma, params->bias, \
+        maybe2half<T>(params->epsilon), params->output, params->skip_input_bias_add_output, \
+        hasBias, hasSkipInputBiasAdditionOutput);                                           \
+    return HIP_CALL(hipPeekAtLastError());                                                  \
```

Comment:
It is better change SkipLayerNormStaticSelection to return type to void,  and check last error after calling SkipLayerNormStaticSelection once. So that we do not duplicate hipPeekAtLastError multiple times in this function.

Reply:
We need Status return type here. SkipLayerNormStaticSelection is used as an instance of TunableOp, and these instances need the same function type `Status(const ParamsT*)`.

Reply:
How about move `hipPeekAtLastError` into `SkipLayerNormKernelSmall` and then just `return SkipLayerNormKernelSmall<...>(...);`

Reply:
Using more strict condition to avoid return hipPeekAtLastError multiple times.